package.json

```diff
@@ -56,5 +56,8 @@
   "browserslist": [
     "> 2%",
     "Last 2 versions"
-  ]
+  ],
+  "dependencies": {
+    "node-sass": "^4.14.1"
```

Comment:
What is this from? I dont think we want to be adding dependencies

Reply:
I believe its already covered in the cmsds scripts package

Reply:
Good pickup, it shouldn't be there. It's just needed for the stand along create-react-app folder.